README.md

```diff
@@ -16,17 +16,63 @@
   <a href="https://nussknacker.io/documentation"><b>Documentation</b></a> &bull;
+  <a href="https://nussknacker.io/documentation/quickstart/lite-streaming/"><b>Quickstart</b></a> &bull;
   <a href="https://nussknacker.io/documentation/docs/installation/"><b>Installation</b></a> &bull;
   <a href="https://cloud.nussknacker.io"><b>Nu Cloud</b></a>  
+
 </h3>
 
 ![image](https://nussknacker.io/documentation/assets/images/nu_designer-87526e47584a5eeb9ce59ad7509d2e7b.png)
 
-Nussknacker lets you design, deploy and monitor real time decision algorithms with easy to use GUI.
+## What is Nussknacker
+
+Nussknacker is a low-code visual tool for domain experts to build, run and monitor real-time decision algorithms instead of implementing them in the code.
+
+In all IT systems, no matter the domain, decisions are made all the time. Which offer should a customer get? Who should receive a marketing message? Does this device need servicing? Is this a fraud?
+
+Algorithms for making such decisions can be developed by programmers. With Nussknacker however, such decision algorithms can be authored and deployed without the need to involve IT.
+
+An essential part of Nussknacker is a visual design tool for decision algorithms. It allows not-so-technical users, like analysts or business people, to author decision logic in an imperative, easy-to-follow and understandable way. Once authored, with a click of a button, scenarios are deployed for execution. And can be changed and redeployed anytime there’s a need.
+
+Nussknacker supports three [processing modes](https://nussknacker.io/documentation/about/ProcessingModes/): streaming, request-response and batch (planned in version 1.15). In streaming mode, Nussknacker uses Kafka as its primary interface: input streams of data and output streams of decisions. In request-response mode, it exposes HTTP endpoints with OpenAPI definitions. And since input events or requests are often not enough to make a decision, Nussknacker can enrich them with data from databases and OpenAPI endpoints.
+
+Nussknacker can be used with its own, lightweight, Kubernetes-based [engine](https://nussknacker.io/documentation/about/engines/). It provides scalability and high availability while being straightforward to set up and maintain. However, in advanced streaming use cases, when aggregations in time windows are needed, Nussknacker can execute decision scenarios on Flink – one of the most advanced platforms for stream processing.
+
+## Why Nussknacker
+
+Nussknacker promises to make developing and deploying real-time decision algorithms as easy as it is with spreadsheets and data at rest. If so, domain experts and developers can focus on tasks that each of these two groups is most happy to perform - domain experts can author the decision algorithms and developers can solve problems beyond the reach of tools like Nussknacker.
+
+We discovered that several factors heavily influence the development of algorithms that work with real-time data:
```

Comment:
IMO each point in the list should have a title that describes what it talks about. 
Eg.
"Low-code"
"Edit-compile-run cycle"
"Testing and debugging"
"Monitoring"
"Production ready"